manifests/addons/s3-blobstore-iam-profile.yml

```diff
@@ -0,0 +1,22 @@
+---
+- type: remove
+  path: /instance_groups/name=bosh/properties/blobstore/access_key_id
```

Comment:
How were these being supplied before?  Were we generating them or passing them in some other way?

Reply:
They didn't exist until my PR the other day.